src/lib.rs

```diff
@@ -60,60 +62,6 @@
 //! ## Optional Features
 //!
 //! - `minor-command` feature enables the commands that are almost never used in modern BMS Players.
-//!
-//! # About the format
-//!
-//! ## Command
-//!
-//! Each command starts with `#` character, and other lines will be ignored. Some commands require arguments separated by whitespace character such as spaces or tabs.
-//!
-//! ```text
-//! #PLAYER 1
-//! #GENRE FUGA
-//! #TITLE BAR(^^)
-//! #ARTIST MikuroXina
-//! #BPM 120
-//! #PLAYLEVEL 6
-//! #RANK 2
-//!
-//! #WAV01 hoge.WAV
-//! #WAV02 foo.WAV
-//! #WAV03 bar.WAV
-//!
-//! #00211:0303030303
-//! ```
-//!
-//! ### Header command
-//!
-//! Header commands are used to express the metadata of the music or the definition for note arrangement.
-//!
-//! ### Message command
-//!
-//! Message command starts with `#XXXYY:ZZ...`. `XXX` is the number of the measure, `YY` is the channel of the message, and `ZZ...` is the object id sequence.
-//!
-//! The measure must start from 1, but some player may allow the 0 measure (i.e. Lunatic Rave 2).
-//!
-//! The channel commonly expresses what the lane be arranged the note to.
-//!
-//! The object id is formed by 2-digit of 36-radix (`[0-9a-zA-Z]`) integer. So the sequence length must be an even number. The `00` object id is the special id, expresses the rest (no object lies). The object lies on the position divided equally by how many the object is in the measure. For example:
-//!
-//! ```text
-//! #00211:0303000303
-//! ```
-//!
-//! This will be placed as:
-//!
-//! ```text
-//! 003|--|--------------|
-//!    |  |03            |
-//!    |  |03            |
-//!    |  |              |
-//!    |  |03            |
-//! 002|--|03------------|
-//!    |  |  []  []  []  |
-//!    |()|[]  []  []  []|
-//!    |-----------------|
-//! ```
```

Comment:
I think the crate root doc comments should be a single truth source of this crate's information. I wrote "About the format" section for the user to focus reading in README, but the README isn't (and shouldn't be) very important as a doc of BMS format. Beacuse of this circumstance, please don't remove this section.